native/src/main/java/io/ballerina/stdlib/azure/functions/Constants.java

```diff
@@ -25,6 +25,8 @@ public interface Constants {
 
     String SLASH = "/";
 
+    String COMMA = ",";
```

Comment:
Let's rename this variable to have a common name representing the purpose of this constant e.g. `ELEMENT_SEPARATOR`
The purpose of defining this as a constant is to change the value to another if the requirement changes. :)